mantidproject/mantid
# Engineering Diffraction GUI GSAS tab improve warnings

There was previously no proper error handling in the GSAS tab of the ED GUI. Errors in loading and refinement (though refinement is not yet properly hooked up) are now dealt with via a message box in the same way as the fitting tab.

**To test:**

Some sample data is [here](https://github.com/mantidproject/mantid/files/1656872/engggui_pr_data.zip).

Open up the ED GUI (`Interfaces -> Diffraction -> Engineering Diffraction`). Go to the GSAS tab and put some nonsense into the **Focused Run** line-edit. Clicking **Load** should give you a message box saying something along the lines of "File doesn't exist".

Click the first **Browse** button and select all the files from the attached zip, then click **Load**. The two runs from 277208 should load fine, and there should be a warning for 277209 that the bank ID was not set.

Partially address #15552 

**Release Notes** 
Will be added to the release notes once refinement can be done from the GUI

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
